src/Domain/Bootstrap.php

```diff
@@ -225,6 +231,33 @@ function() {
 		);
 	}
 
+	/**
+	 * Load and set up the Interactivity API if enabled.
+	 */
+	protected function load_interactivity_api() {
+	// phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
+		$is_enabled = apply_filters(
+			'woocommerce_blocks_enable_interactivity_api',
+			true
+		);
+
+		if ( $is_enabled ) {
+			require_once '../Interactivity/load.php';
```

Comment:
I was able to make it work after following change:
```suggestion
			require_once __DIR__ . '/../Interactivity/load.php';
```